great_ape_safe/ape_api/curve.py

```diff
@@ -31,56 +32,75 @@ def deposit(self, lp_token, mantissas, asset=None):
         # TODO: find and route through zaps automatically
         # TODO: could pass dict to mantissas with {address: mantissa} and sort
         #       out proper ordering automatically?
-        pool_addr = self.registry.get_pool_from_lp_token(lp_token)
-        pool = self.safe.contract(pool_addr)
+        
+        # pool must be provided; cannot look up v2 factory pools from lp token
```

Comment:
looks like `lp_token.minter()` should do the trick:
```
>>> from helpers.addresses import registry
>>> lp_token = interface.ICurveLP(registry.eth.treasury_tokens.badgerWBTC_f)
>>> lp_token.minter()
'0x50f3752289e1456BfA505afd37B241bca23e685d'
>>> registry.eth.crv_factory_pools.badgerWBTC_f
'0x50f3752289e1456BfA505afd37B241bca23e685d'
```

Reply:
forgot that method existed, very helpful